datafusion/expr/src/built_in_function.rs

```diff
@@ -69,14 +69,10 @@ pub enum BuiltinScalarFunction {
     Cos,
     /// cos
     Cosh,
-    /// Decode
```

Comment:
This is the key point of this PR: to begin removing items from this (giant) enum